docs/paper/admin/reference/configuration/spigot-configuration.mdx

```diff
@@ -0,0 +1,563 @@
+      sweetberry-modifier:
+        default: '100'
+        description: 'The growth modifier percentage, where Vanilla is 100%.'
+      torchflower-modifier:
+        default: '100'
+        description: 'The growth modifier percentage, where Vanilla is 100%.'
+      twistingvines-modifier:
+        default: '100'
+        description: 'The growth modifier percentage, where Vanilla is 100%.'
+      vine-modifier:
+        default: '100'
+        description: 'The growth modifier percentage, where Vanilla is 100%.'
+      weepingvines-modifier:
+        default: '100'
+        description: 'The growth modifier percentage, where Vanilla is 100%.'
+      wheat-modifier:
+        default: '100'
+        description: 'The growth modifier percentage, where Vanilla is 100%.'
+    hanging-tick-frequency:
+      default: '100'
+      description: 'How often to tick hanging entities, in ticks.'
+    hopper-amount:
+      default: '1'
+      description: 'How many items a hopper should move at a time, limited to stack size.'
+    hopper-can-load-chunks:
+      default: 'false'
+      description: 'Whether to prevent hoppers from loading chunks.'
+    hunger:
+      combat-exhaustion:
+        default: '0.1'
+        description: 'How much exhaustion to give from attacking.'
+      jump-sprint-exhaustion:
+        default: '0.2'
+        description: 'How much exhaustion to give from sprint jumping.'
+      jump-walk-exhaustion:
+        default: '0.05'
+        description: 'How much exhaustion to give from jump walking.'
+      other-multiplier:
+        default: '0.0'
+        description: 'The exhaustion multiplier for normal walking and crouching.'
+      regen-exhaustion:
+        default: '6.0'
+        description: 'How much exhaustion to give from regenerating health.'
+      sprint-multiplier:
+        default: '0.1'
+        description: 'The exhaustion multiplier for sprinting on the ground.'
+      swim-multiplier:
+        default: '0.01'
+        description: 'The exhaustion multiplier for when "swimming".
+          This occurs when the player is:
+
+            - actually swimming
+
+            - walking underwater (eyes below surface)
+
+            - walking on water (eyes above surface)'
+    item-despawn-rate:
+      default: '6000'
+      description: 'The default time in ticks it takes an item to despawn.
+        Per-item rates are found in Paper''s [per-world config](/paper/reference/world-configuration).'
+    max-tick-time:
+      entity:
+        default: '50'
+        description: 'This option does not operate, and is non-functional upstream.'
+      tile:
+        default: '50'
+        description: 'This option does not operate, as it is disabled by a Paper patch.'
+    max-tnt-per-tick:
+      default: '100'
+      description: 'How many TNT to process per server tick. Set to 0 to disable.'
+    merge-radius:
+      exp:
+        default: '3.0'
+        description: 'The range, in blocks, that exp orbs will combine at.'
```

Comment:
It's important to say that this logic is 100% custom and doesn't exists in vanilla and that vanilla merge radius is unchanged once the orb is spawned